engine/polyglot-api/src/main/java/org/enso/polyglot/RuntimeOptions.java

```diff
@@ -137,6 +142,7 @@ public class RuntimeOptions {
               PREINITIALIZE_DESCRIPTOR,
               WAIT_FOR_PENDING_SERIALIZATION_JOBS_DESCRIPTOR,
               USE_GLOBAL_IR_CACHE_LOCATION_DESCRIPTOR,
+              ENABLE_ASSERTIONS_DESCRIPTOR,
```

Comment:
Having a runtime option for `EnsoLanguage` & `EnsoContext` is fine, but...

Reply:
Removed in b435b75360098870a8973ac600c81472ec623d2e